kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -311,51 +311,43 @@ def run(self, labels: list, analysis_id=None):
 
                 with open(".thoth.yaml", "r") as f:
                     thoth_config = yaml.safe_load(f)
-
-                if thoth_config.get("overlays_dir"):
-                    runtime_environments = [
-                        e["name"] for e in thoth_config["runtime_environments"]
-                    ]
-                else:
-                    runtime_environments = [
-                        thoth_config["runtime_environments"][0]["name"]
-                    ]
```

Comment:
why is part not needed anymore?

are we checking somewhere else, if the repo is overlays or normal type?

Reply:
Yes, this code block is moved to https://github.com/thoth-station/kebechet/blob/c54451adaabe27190a749354ce0acb7a414cf339/kebechet/kebechet_runners.py#L176 to remove code redundancy

Reply:
sounds good. thanks